throwing-rabish.md

```diff
@@ -0,0 +1 @@
+# Throwing rabish
```

Comment:
Be careful , you spell wrong your file s title, do not put space in it.
any way, I will be happy if you can make more request to comment